src/extension.ts

```diff
@@ -320,50 +324,45 @@ function execWorkspaceCommand(client: LanguageClient, params: ExecuteCommandPara
 	return client.sendRequest(ExecuteCommandRequest.type, params);
 }
 
-interface module {
+interface moduleCaller {
 	uri: string,
-	name: string
+	rel_path: string
 }
 
 interface moduleCallersResponse {
-	moduleCallers: module[],
-	needsInit: boolean
+	v: number,
```

Comment:
Is this the API version? Even though it's not being referenced, let's give it a more descriptive name.

Reply:
yep, updated.